fineract-provider/build.gradle

```
@@ -567,3 +576,11 @@ tasks.withType(SpotBugsTask) {
     reports.html.enabled = true
     reportLevel = "high"
 }
+
+gitProperties {
+    gitPropertiesResourceDir = "$buildDir/classes/java/main/resources"
+    dateFormat = "yyyy-MM-dd'T'HH:mmZ"
+    dateFormatTimeZone = "GMT"
+}
+
+generateGitProperties.outputs.upToDateWhen { false }
```

May be it would be useful to document what this does inline? According to https://github.com/n0mer/gradle-git-properties/blob/master/README.md#other-usages:

```suggestion
// make sure the generateGitProperties task always executes (even when git.properties is not changed)
generateGitProperties.outputs.upToDateWhen { false }
```